doc/quickstart/find_data.rst

```diff
@@ -564,6 +564,75 @@ explained in :ref:`extra_facets`, and which content is :download:`available here
 </../esmvalcore/config/extra_facets/ipslcm-mappings.yml>`. These multi-variable
 files must also undergo some data selection.
 
+.. _read_access-esm:
+
+ACCESS-ESM
+^^^^^^^^^^
+
+ESMValTool can read native `ACCESS-ESM <https://research.csiro.au/access/about/esm1-5/>`__
+model output.
+
+.. warning::
+
+  This is the first version of ACCESS-ESM CMORizer for ESMValCore Currently, 
+  Supported variables: ``pr``, ``ps``, ``psl``, ``rlds``, ``tas``, ``ta``, ``va``,
+  ``ua``, ``zg``, ``hus``, ``clt``, ``rsus``, ``rlus``.
+
+This is an example of the directory file structure in `config_developer.yml`.
+
+.. code-block:: yaml
+
+  ACCESS:
+    cmor_strict: false
+    input_dir:
+      default:
+        - '{institute}/{sub_dataset}/{exp}/{modeling_realm}/netCDF'
+    input_file:
+      default: '{sub_dataset}.{special_attr}-*.nc'
+    output_file: '{institute}_{sub_dataset}_{special_attr}_{short_name}'
+    cmor_type: 'CMIP6'
+    cmor_default_table_prefix: 'CMIP6_'
+
+.. hint::
+
+  We only provide one default `input_dir` since this is how ACCESS-ESM native data was 
+  stored on NCI. Users can modify this path to match their local file structure.
+
+
+Thus, example dataset entries could look like this:
+
+.. code-block:: yaml
+
+  dataset:
+    - {project: ACCESS, institute: ACCESS-ESM1-5, mip: Amon, dataset:ACCESS_ESM, sub_dataset: HI-CN-05, 
+      exp: history, modeling_realm: atm, special_attr: pa, start_year: 1986, end_year: 1986}
+
+
+`dataset` and `sub_dataset` are not redundant, `dataset` is for ESMValCore to search for CMORizer,
+which is always `ACCESS_ESM`, `sub_dataset` is dataset under root `ACCESS-ESM1-5`.
+
+`special_attr` is a special attribute in the filename `ACCESS-ESM` raw data, it's related to frquency
+of raw data.
+
+`modeling_realm` is a realm attribute, it include `atm`, `ice` and `oce`.
```

Comment:
Please move these two and an entry for `sub_dataset` to the table. These are also extra facets.